ceno_zkvm/src/instructions/riscv/branch/b_circuit.rs

```diff
@@ -0,0 +1,148 @@
+use std::marker::PhantomData;
```

Comment:
Suggest 
1. we can combine another circuit "ceno_zkvm/src/instructions/riscv/branch/beq_circuit.rs" into this file follow the same methodology.
2. then the ultimate filename can be `branch_circuit.rs`, follow another convention e.g. "logic_circuit.rs", "logic_imm_circuit.rs"
3. we can swap `(is_signed_lt, is_unsigned_lt, branch_taken_bit)` order to `(branch_taken_bit, is_signed_lt, is_unsigned_lt)` so we dont need to have extra `clone` on config
4. at last, we can remove `blt.rs` and `bltu.rs`
5. (not related to this PR, but better to have) please add few simple unittest since previously those are missing.

Reply:
Just to clarify, more tests are required than what is there in the `src/instructions/riscv/branch/test.rs` file?

Reply:
ohoh I noticed we already have tests in `src/instructions/riscv/branch/test.rs`
So please ignore my point 5 above 😺